### PR TITLE
[4.0] re-do temporary change of filter published-condition

### PR DESCRIPTION
**Pull request by chmst**
### Summary of Changes
As required in #30467 the filter.condition is changed to filter.published.

### Testing Instructions
As in #30445
In a module "Articles latest" only published items my be displayed.

### Documentation Changes Required
no

### CODE REVIEW

File: modules/mod_articles_news/src/Helper/ArticlesNewsHelper.php [hunk 1]
```diff
@@ -50,7 +50,7 @@ public static function getList(&$params)
 		$model->setState('params', $appParams);
 
 		$model->setState('list.start', 0);
-		$model->setState('filter.condition', ContentComponent::CONDITION_PUBLISHED);
```
Comment (Quy) [inline modules/mod_articles_news/src/Helper/ArticlesNewsHelper.php:53]: Remove use statement on line 21.

**Comment by maikol-ortigueira** [comment]
I can not replicate this.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31155">issues.joomla.org/tracker/joomla-cms/31155</a>.</sub>

**Comment by gostn** [comment]
I have tested this item :white_check_mark: successfully on f97121c638f57fc044cd93ee84239f4ab9f500ed<br /><br />Only published, no unpublished or trashed articles are shown.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31155">issues.joomla.org/tracker/joomla-cms/31155</a>.</sub>

**Comment by particthistle** [comment]
I have tested this item :white_check_mark: successfully on f97121c638f57fc044cd93ee84239f4ab9f500ed<br /><br />Tested successfully. Thanks @chmst.

The code is a refactoring of the filter selector, so no change to what appears is expected for the test case. Tried various ordering options for the module, all were successful.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31155">issues.joomla.org/tracker/joomla-cms/31155</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31155">issues.joomla.org/tracker/joomla-cms/31155</a>.</sub>

**Comment by richard67** [comment]
@alikon RTC with only 1 test? Or have you tested, too, and just forgotten to submit the test result?

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 9702057f64ce0eeb1d9f9f1d328da2245fc52e85<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31155">issues.joomla.org/tracker/joomla-cms/31155</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31155">issues.joomla.org/tracker/joomla-cms/31155</a>.</sub>

**Comment by richard67** [comment]
Previous test by @particthistle was still valid because the commit after that test was just a clean branch update.

I've restored the test result in the issue tracker so it's correctly counted there and here.

**Comment by richard67** [comment]
Thanks!

**Pull request merged**